packages/rollup/test/integration/BUILD.bazel

```diff
@@ -0,0 +1,45 @@
+load("@build_bazel_rules_nodejs//internal/golden_file_test:golden_file_test.bzl", "golden_file_test")
+load("@npm_bazel_rollup//:index.from_src.bzl", "rollup_bundle")
+
+_BUNDLE_FORMATS = [
+    "amd",
```

Comment:
do we really need coverage of all of them?

I can't imagine given our thin layering on top of rollup that we'd have a bug in only one format

Reply:
hmmmmm. I think the point was more to validate that all the possible values in the attribute constraints
```
values = ["amd", "cjs", "esm", "iife", "umd", "system"],
```
produced the expected bundle types. Verified that manually by looking at them and golden will validate that they continue to do so.

However, on that note, having the attribute constrained when rollup is a peerDependency seems wrong as well. If a newer version of rollup adds another bundle type in the future it won't be possible to use that without changing the constraints here. The constraints do have the benefit of validating that the user doesn't accidentally type `es` instead `ems` (for example) however it doesn't seem like the scope of the rule to prevent user typos and if they are expecting `ems` output and get `cjs` instead then they should notice on their end.

Reply:
yeah maybe we shouldn't constrain them - but we should give a better error message cuz rollup will tell you to change the wrong thing

in theory we want to constrain when we add the JSModuleInfo provider so everyone is supposed to agree on the options

I think since we have the field at all, it's nice to duplicate the Rollup validation rules on it.

Reply:
ok. sounds like that decision is out of the scope of this PR so lets discuss elsewhere